homeassistant/components/fritzbox/manifest.json

```diff
@@ -8,7 +8,7 @@
       "st": "urn:schemas-upnp-org:device:fritzbox:1"
     }
   ],
-  "codeowners": [],
+  "codeowners": ["@mib1185"],
```

Comment:
Your PR miss a tick in:
The manifest file has all fields filled out correctly.
Updated and included derived files by running: python3 -m script.hassfest.

Reply:
I run `python3 -m script.hassfest` before commit... What did you mean, what is wrong?

Reply:
Nothing is wrong, but if you look at your PR text, you did not checkmark, that you did run the script. Which confused me.

"If the code communicates with devices, web services, or third-party
[]  The manifest file has all fields filled out correctly.
Updated and included derived files by running: python3 -m script.hassfest."


Thanks a lot for the hint, how simple when you know how to.